tritium-metrics-jvm/src/main/metrics/metrics.yml

```diff
@@ -37,3 +37,10 @@ namespaces:
       attribute.uptime:
```

Comment:
In a follow up we should probably get rid of `attribute.name` and `attribute.vendor` since they're provided by our uptime gauge.

Reply:
actually have just tacked it on here, as we're conceptually moving this info from a `Gauge<String>` (which isn't representable in DD) to tags on the numerical uptime gauge.

Reply:
I like it. We should probably flag this as a break, despite even though it could only break manual workflows.